core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java

```diff
@@ -1695,6 +1695,8 @@ private boolean rerunWF(String workflowId, String taskId, Map<String, Object> ta
             rerunFromTask.setUpdateTime(0);
             rerunFromTask.setEndTime(0);
             rerunFromTask.setOutputData(null);
+            rerunFromTask.setRetried(false);
```

Comment:
Is this change related to disabling background workers?

Reply:
no, this is just some required cleanup.